legacy_packages/react/src/wallet/wallets/embeddedWallet/CountrySelector.tsx

```diff
@@ -1,1271 +1,9 @@
+import { useQuery } from "@tanstack/react-query";
 import React, { useRef } from "react";
 import { Theme, fontSize, spacing } from "../../../design-system";
 import { useCustomTheme } from "../../../design-system/CustomThemeProvider";
 import { StyledSelect } from "../../../design-system/elements";
 
-const placeholderSupportedSmsCountries = [
```

Comment:
@ElasticBottle Can we bring back the old list of countries with full country name and show the full country name in the UI? 

Currently we only show the ISO code - which is kinda hard for users to find the correct one for their country